dino-code/app/components/ChallengeCarousel.module.css

```diff
@@ -0,0 +1,161 @@
+.carouselContainer {
+  width: 100%;
+  max-width: 350px;
+  margin: 0 auto;
+  position: relative;
+  touch-action: pan-y; /* Enable vertical scrolling but disable horizontal to prevent conflicts */
+}
+
+.carouselWrapper {
+  position: relative;
+  overflow: hidden;
+  border-radius: 13px;
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+  user-select: none; /* Prevent text selection during swipe */
+}
+
+.carouselTrack {
+  display: flex;
+  transition: transform 0.3s ease-in-out;
+  width: 100%;
+  height: 100%;
+}
+
+.carouselSlide {
+  width: 100%;
+  flex-shrink: 0;
+}
+
+.card {
+  background-color: white;
+  border-radius: 20px;
+  overflow: hidden;
+  margin: 0 auto;
+  width: 100%;
+}
+
+.cardHeader {
+  padding: 12px 16px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start; /* Align content to the left */
+  width: 100%; /* Full width of the card */
+}
+
+.headerContent {
```

Comment:
Does this belong to card? How do I know? Pls use naming convention. Consider naming it `cardHeader__content`.